src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PopupWindow.cs

```diff
@@ -132,7 +139,15 @@ private Subview ActiveView
         private PopupViewType ActiveViewType
         {
             get { return activeViewType; }
-            set { activeViewType = value; }
+            set
+            {
+                if (activeViewType != value)
+                {
+                    ActiveView.OnDisable();
+
```

Comment:
Let's not have all this wasted vertical space, I don't have that much space on my laptop (i.e. totally redundant empty line)